### ISSUE TITLE
R&D images combining

### ISSUE THREAD

**Issue by Astrogem2** [tgstation/tgstation#81478]
Reporting client version: 515.1630

## Issue Summary
When using the R&D console, the icons in the technology nodes combine whenever I select select another node when I'm already looking at one.

## Round ID:
[223953](https://scrubby.melonmesa.com/round/223953)

## Testmerges:
- [Temp keybinds patch for 515.1628](https://www.github.com/tgstation/tgstation/pull/81089)
- [A red spy has entered the base: Adds Spies, a roundstart antagonist inspired by Goonstation's Spy-Thief](https://www.github.com/tgstation/tgstation/pull/81231)

## Reproduction:

Basically, when using the R&D console, when you get the details of a node and then get the details of another node by looking at the future research of it or the needed research, it adds it to the node.
![image_2024-02-14_165140655](https://github.com/tgstation/tgstation/assets/101240878/beba27f4-6a72-4745-b588-414db52675ea) (clearly not meant to happen)
